x/appchain/common/types/shared_params.go

```diff
@@ -0,0 +1,46 @@
+package types
+
+import (
+	"fmt"
+	"time"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+const (
+	// DefaultIBCTimeoutPeriod is the default timeout period for IBC packets.
+	DefaultIBCTimeoutPeriod = 4 * 7 * 24 * time.Hour
```

Comment:
It will influence the unbonding duration, but not directly. It is the worst case timeout period. If a subscriber chain goes offline, this timeout comes into play. Otherwise, the unbonding duration is just the time to the end of the epoch + subscriber's unbonding duration + the time it takes for message delivery.

Each packet sent from the coordinator to the subscriber has this timeout period set. If the subscriber doesn't process the packet within this period, it is considered offline and removed as an AVS. During its removal, all associated unbondings are freed.